diff --git a/dist/component/convex.config.js b/dist/component/convex.config.js
index 3d15ed6b9a71d4f2e29f846cd61b4d85531ff13a..d6180eabf7eef4c69f22026ad14e9cbee8c1280e 100644
--- a/dist/component/convex.config.js
+++ b/dist/component/convex.config.js
@@ -1,6 +1,13 @@
 import { defineComponent } from "convex/server";
+import { v } from "convex/values";
 import workpool from "@convex-dev/workpool/convex.config";
-const component = defineComponent("agentmail");
+const component = defineComponent("agentmail", {
+    env: {
+        AGENTMAIL_API_KEY: v.optional(v.string()),
+        AGENTMAIL_BASE_URL: v.optional(v.string()),
+        AGENTMAIL_WEBHOOK_SECRET: v.optional(v.string()),
+    },
+});
 component.use(workpool, { name: "sendPool" });
 component.use(workpool, { name: "callbackPool" });
 export default component;
